client/jetpack-cloud/sections/pricing/sale-banner/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import React, { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import type { JetpackSaleCoupon } from 'calypso/state/marketing/selectors';
+
+import './style.scss';
+
+const getTimeLeftFromSecondsLeft = ( timeDiff: number ) => {
+	return {
+		days: Math.floor( timeDiff / ( 60 * 60 * 24 ) ),
+		hours: Math.floor( ( timeDiff / ( 60 * 60 ) ) % 24 ),
+		minutes: Math.floor( ( timeDiff / 60 ) % 60 ),
+		seconds: Math.floor( timeDiff % 60 )
+			.toString()
+			.padStart( 2, '0' ),
+	};
+};
+
+type Props = {
+	coupon: JetpackSaleCoupon;
+};
+
+const SaleBanner: React.FC< Props > = ( { coupon } ) => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+	const moment = useLocalizedMoment();
+	const [ isClosed, setIsClosed ] = useState( false );
+	const now = moment.utc().unix();
+	const expiryDate = moment.utc( coupon?.expiry_date ).unix();
+	const isBeforeExpiry = coupon && now <= expiryDate;
+	const [ timeLeft, setTimeLeft ] = useState( getTimeLeftFromSecondsLeft( expiryDate - now ) );
+	const { days, hours, minutes, seconds } = timeLeft;
+
+	useEffect( () => {
+		const intervalId = setInterval(
+			() => setTimeLeft( getTimeLeftFromSecondsLeft( expiryDate - now ) ),
+			1000
+		);
+
+		return () => clearInterval( intervalId );
+	}, [ timeLeft, expiryDate, now, setTimeLeft ] );
+
+	const closeBanner = () => {
+		dispatch( recordTracksEvent( 'calypso_pricing_page_sale_banner_close_button_click' ) );
+		setIsClosed( true );
+	};
+
+	return (
+		<>
+			{ ! isClosed && isBeforeExpiry && (
+				<div className="sale-banner" role="banner" aria-label={ translate( 'Discount Banner' ) }>
+					<div className="sale-banner__content">
+						<div>
+							<b>{ translate( 'End of Summer Sale!' ) }</b>
+							&nbsp;
+							{ translate( 'Get %(discount)d%% off your first year of Jetpack', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Get %d%% off your first year of Jetpack' )` *ES Score: 6*